src/components/image/index.test.tsx

```diff
@@ -55,4 +57,40 @@ describe('Images', () => {
       'whatImg'
     );
   });
+  describe('Cloudinary', () => {
+    test('should have the correct className and publicId', () => {
+      const CLASS_NAME_TEST = 'feed-item__image';
+      const PUBLIC_ID_TEST = 'QmRLG913uKX7QxwSFMk1TMhtjxwy6kVek37HTcR7AtJUVf';
+
+      const wrapper = subject({
+        className: CLASS_NAME_TEST,
+        src: `https://ipfs.fleek.co/ipfs/${PUBLIC_ID_TEST}`,
+        useCloudinary: true,
+      });
+
+      expect(wrapper.find(AdvancedImage).prop('className')).toEqual(
```

Comment:
if you use `expect([element].hasClass(CLASS_NAME_TEST)).toBe(true);` it will make this a bit less brittle. eg. .. this will fail if we add any other classes to the underlying element, which shouldn't matter from the perspective of this test. all we care is that the class we give it is being applied.